examples/apps/echo/workload/echo_factory.cpp

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
No need of echo_factor.h and echo_factory.cpp. These could be moved to echo.h. EchoResultImpl  in echo.h is not required. Process in EchoResultImpl  can be a private function in EchoResult. 
Similarly no need of heart_disease_evaluation_factory.cpp

Reply:
Ok